internal/cri/server/container_status.go

```diff
@@ -34,6 +34,9 @@ import (
 func (c *criService) ContainerStatus(ctx context.Context, r *runtime.ContainerStatusRequest) (*runtime.ContainerStatusResponse, error) {
 	container, err := c.containerStore.Get(r.GetContainerId())
 	if err != nil {
+		if errdefs.IsNotFound(err) {
+			return nil, err
```

Comment:
What's the reason for changing the error here?

Reply:
Pass the `NotFound` error type to the caller which is instrument